docs/get-started/on-device-training.md

```diff
@@ -0,0 +1,49 @@
+---
+title: On-Device Training
+parent: Get Started
+nav_order: 12
+---
+
+# On-Device Training with ONNX Runtime
+{: .no_toc }
+`On-Device Training` refers to the process of training a model on an edge device, such as mobile phones, embedded devices, gaming consoles, web browsers, etc. This is in contrast to training a model on a server or a cloud. Training on the device
+
+- Can improve data privacy and security, especially when working with sensitive data that cannot be shared with a server or a cloud.
+- It is also useful for the task of personalization where the model needs to be trained on the user's data.
+- It can also be beneficial in scenarios where network connectivity is unreliable or limited.
+
+In addition to the above scenarios, there could be many more that warrant the need for `On-Device Training`.
+
+
+`ONNX Runtime Training` offers an easy way to efficiently train and infer a wide range of ONNX models on edge devices. The training process is divided into two phases:
+- [the offline phase](#the-offline-phase)
+- [the training phase](#the-training-phase).
+
+## The Offline Phase
```

Comment:
Good to call out that this phase does not use user data on the cloud.

Reply:
Shall we call this Training Artifact Generation Phase? In my experience, "offline phase" is a bit confusing to some people.